src/background/controllers/onInstallController.ts

```diff
@@ -0,0 +1,55 @@
+import QryptoController from '.';
+import IController from './iController';
+
+export default class OnInstallController extends IController {
+  constructor(main: QryptoController) {
+    super('onInstall', main);
+
+    /*
+    * When the extension is installed or updated, refresh the tabs of all
+    * dapp apps using Qrypto. We want to refresh these tabs so that they can
+    * have a clean slate for adding event listeners and injecting scripts,
+    * and so that we do not have any dangling/duplicate event listeners or
+    * injected scripts from prior versions of Qrypto.
+    * To get refreshed, dapp tabs must implement the
+    * handleQryptoInstallOrUpdate event listener described in the Readme.
+    */
+    chrome.runtime.onInstalled.addListener((details) => {
+      if (details.reason === 'install' || details.reason === 'update') {
+        this.refreshAllDappTabs();
+      }
+    });
+
+    this.initFinished();
+  }
+
+  public refreshAllDappTabs = () => {
+    // Get all windows
+    chrome.windows.getAll({
+      populate: true,
+    }, (windows) => {
+        for (const currentWindow of windows ) {
+          if (currentWindow.tabs) {
+            for (const currentTab of currentWindow.tabs ) {
```

Comment:
remove space in bracket

Reply:
hmm my linter doesnt seem to fix this, is it supposed to?

Reply:
ya I would've thought it would. can you see if theres a rule for that?

Reply:
looks like it should be this rule: https://eslint.org/docs/rules/space-in-parens